extconf.rb

```diff
@@ -132,11 +132,11 @@ def rule(target, clean = nil)
 else
    version = 0
 end
-if version < 73
+if version < 92
```

Comment:
If we drop support for anything before 9.2 we should also remove the check against 7.4 below in extconf.rb.

Reply:
You should also update the README if we increase the max version.

Reply:
Agreed, but removing everything `< 9.2` will be pretty huge cleanup in all the codebase.  That said, it is something I'd like to do - but not here in #part1 (nor #part2) - the major thing is to allow successful build of plruby and make the testsuite pass again (start the Travis CI, etc.).

Reply:
Fair enough. It makes sense to do the clean up in a separate in a commit.